### PR TITLE
Move timeouts and rate-limiting to separate docs chapter

### PR DESCRIPTION
**Pull request by JOJ0**
What do you think?

Any better ideas for a chapter name? Advanced Configuration? Settings? Advanced Settings?

What I don't like is that it looks pretty similar to "Contribution" since that is right next to it. Maybe just another position in the menu would fix that?

Suggestions welcome.
Cheers!

### CODE REVIEW

**Comment by JOJ0** [comment]
image

**Comment by AnssiAhola** [comment]
Would it make any difference if we renamed `Contribution` to `Contributing`?

image

**Review by prcutler — COMMENTED** [review]
A couple notes:

I agree that it should be "Contributing".  I also like the idea of moving it up after Quickstart and before Authentication.

I'd like to reword line 22, something like:

`For example, to enable a request timeout of 5 seconds:` 

or something like that, because right after the example you did a great job of explaining what values are supported.  At a minimum, I would recommend adding a colon.

**Comment by alifhughes** [comment]
My 2 cents..

I agree with "Contributing". As for the configuration title, I was thinking as both as these are optional configurations, we could call it "Optional Configuration"? 

I would keep it after "Authentication" though as auth is one of the most common questions/issues/requests we've had. 

Other than thatm I agree with @prcutler's rewording too. 

Hope that helps!

**Comment by JOJ0** [comment]
https://python3-discogs-client.readthedocs.io/en/docs_config_chapter/

have a look. Enabled the branch (hidden just with link).

What do you think? Kept the position but renamed to optional. Reworded the "timeout sentence".

File: docs/source/index.rst [hunk 1]
```diff
@@ -16,8 +16,8 @@ python3-discogs-client documentation
    fetching_data.md
    fetching_data_repl.md
    listing.md
-   requests_rate_limit.md
-   contribution.md
+   configuration.md
```
Comment (AnssiAhola) [inline docs/source/index.rst:19]: To nitpick :sweat_smile: , should this be `optional_configuration.md` ?

**Comment by JOJ0** [comment]
Thanks for the hint Anssi! Fixed!

File name was intentionally just configuration.md instead of longer, but you're right, probably it could be confusion when quickly trying to find it by beginning later. Will fix that and merge!

**Review by AnssiAhola — APPROVED** [review]
LGTM! :+1:

**Pull request merged**